src/components/menu/menu.ts

```diff
@@ -22,15 +22,11 @@ import { Popover, PopoverProps } from "../popover/popover";
 
 css/* scss */ `
   .o-menu {
-    background-color: white;
```

Comment:
Having a `background-color: white;` instead of bootstrap `bg-white` is actually intentional, because `bg-white` is overwritten in dark mode